python/foglamp/plugins/south/cc2650/cc2650.py

```diff
@@ -0,0 +1,417 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for Sensortag CC2650 plugin """
+
+import copy
+import time
+import uuid
+
+from foglamp.common import logger
+from foglamp.plugins.common import utils
+from foglamp.plugins.south.cc2650.sensortag_cc2650 import *
+from foglamp.services.south import exceptions
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_DEFAULT_CONFIG = {
+    'plugin': {
+        'description': 'TI SensorTag South Plugin',
+        'type': 'string',
+        'default': 'cc2650',
+        'readonly': 'true'
+    },
+    'bluetoothAddress': {
+        'description': 'Bluetooth address',
+        'type': 'string',
+        'default': 'B0:91:22:EA:79:04',
+        'order': '1'
+    },
+    'assetNamePrefix': {
+        'description': 'Asset prefix',
+        'type': 'string',
+        'default': 'CC2650/%M/',
+        'order': '2'
+    },
+    'pollInterval': {
+        'description': 'The interval between poll calls to the SensorTag poll routine expressed in milliseconds.',
+        'type': 'integer',
+        'default': '1000',
+        'order': '3'
```

Comment:
Please make unit seconds and default 1. Also make sure south server is able to handle 0

Reply:
let's handle it with FOGL-1780